released/terraform-be/main.tf

```
@@ -0,0 +1,35 @@
+terraform {
+  required_providers {
+    btp = {
+      source  = "sap/btp"
+      version = "1.3.0"
+    }
+    cloudfoundry = {
+      source = "SAP/cloudfoundry"
+      version = "0.1.0-beta"
+    }
+  }
+}
+
+provider "btp" {
+  globalaccount  = "terraformintcanary"
```

Canary might be confusing for our customers 

```suggestion
  globalaccount  = "<GLOBALACCOUNT SUBDOMAIN>"
```